diff --git a/build/globals/SteamClient.d.ts b/build/globals/SteamClient.d.ts
index 2357077560c52548b36c727d0fb759faf5be2d44..9a1fb61cb25d8fe7c1ca7ecba23b9f114962c501 100644
--- a/build/globals/SteamClient.d.ts
+++ b/build/globals/SteamClient.d.ts
@@ -1,6 +1,3 @@
-declare global {
-    var SteamClient: SteamClient;
-}
 export interface Apps {
     RegisterForAppOverviewChanges: any;
     RegisterForAppDetails: any;
diff --git a/src/globals/SteamClient.ts b/src/globals/SteamClient.ts
index 0c65707782a81f5975b25f52eec5655f8abebe28..24ed6075372188c745c505523e1e5b938e216aef 100644
--- a/src/globals/SteamClient.ts
+++ b/src/globals/SteamClient.ts
@@ -1,7 +1,3 @@
-declare global {
-  var SteamClient: SteamClient;
-}
-
 export interface Apps {
   RegisterForAppOverviewChanges: any;
   RegisterForAppDetails: any;
